src/sagas/account.js

```diff
@@ -27,24 +27,22 @@ import { getTokenName } from '../selectors/env';
 import { minDelayCall } from './helpers';
 
 export function * doSignup (action) {
-  const { formData, history } = action;
+  const { formData } = action;
   try {
     const response = yield minDelayCall(api.signup, formData);
 
     // Persist token for future page loads
     const tokenName = yield select(getTokenName);
     new Cookies().set(tokenName, response.key, { path: '/' });
 
-    // Redirect user to home
-    history.replace('/');
```

Comment:
So, if we want this behavior, we need to create a middleware to listen for the proper action and do the redirect ourselves?

Reply:
Correct, I just added this to the rootSaga of my app like so:
```
   ...
    // Wait for login or load user
    yield take([LOGIN_SUCCESS, LOAD_USER_SUCCESS]);

    // Redirect user to home
    navigation.navigate('Home');
    ...
```
The reason for this is because I am using the library for both `react` and `react-native` apps which use different methods of navigation. Instead of adding some complex functionality to the core library to handle this I just transferred the responsibility of redirecting after login to the app itself.

Reply:
:+1: